src/components/ProgramList.tsx

```diff
@@ -0,0 +1,177 @@
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "blended",
+            tag: "music",
+        },
+        {
+            image: "https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all&zoom=1&resize=644%2C428",
+            name: "MPM/JELIC",
+            description:
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "online",
+            tag: "math",
+        },
+        {
+            image: "https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all&zoom=1&resize=644%2C428",
+            name: "Education Through Creativity",
+            description:
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "online",
+            tag: "math",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+    ];
+
+    return (
+        <Center>
+            <Wrap spacing="50px" justify="space-between">
+                {imagesAndDescriptions.map((item, idx) => {
+                    return item.name == "placeholder" ? (
+                        <WrapItem
+                            flexBasis="300px"
+                            flexGrow={1}
+                            display="hidden"
+                            key={idx}
+                        ></WrapItem>
+                    ) : (
+                        <WrapItem flexBasis="300px" key={idx} flexGrow={1}>
+                            <Box
+                                borderWidth="1px"
+                                width="100%"
+                                _hover={{
+                                    borderColor: "#0C53A0",
+                                    borderWidth: 1,
+                                }}
+                                onClick={() => {
+                                    if (!session)
+                                        window.location.href = "/login";
+                                    // for testing (/programdetails is in #46)
+                                    else
+                                        window.location.href =
+                                            "/programdetails" + idx;
```

Comment:
Will this be changed later?

This is the index in the list right?What if the index doesn't correspond to the id?

Reply:
yeah its for testing, its meant to represent the id
i also made the test data in #46 a hashmap of (0 or 1) -> test data, so we can click the first 2 cards....